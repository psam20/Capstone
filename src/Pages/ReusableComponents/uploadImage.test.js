import React from 'react';
import {shallow} from 'enzyme';
import UploadImage from './uploadImage';

describe('Upload Image Component Test Case',()=>{
    let props={
        onChange:jest.fn(),
        image:{
            preview:'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcSq9_coI2dlwltnSCwfiuoIrTYKFQb4of1Rf-2VgAKpGr00TDd8&usqp=CAU'
        }
    }
    let wrapper=shallow(<UploadImage {...props}/>)
     let input=wrapper.find('input');
   
    it('Upload Image Component should Rendered',()=>{
        expect(wrapper).toMatchSnapshot();
    })
    it('Upload Image Should Contain Input TAG',()=>{
        expect(input.length).toBe(1);
    })
})
