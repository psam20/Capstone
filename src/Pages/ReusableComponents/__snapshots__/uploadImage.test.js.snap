// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload Image Component Test Case Upload Image Component should Rendered 1`] = `ShallowWrapper {}`;
